i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/how_to_write_plugin_dataproxy.md

```diff
@@ -0,0 +1,100 @@
+---
+title: DataProxy 插件
+sidebar_position: 6
+---
+
+## 总览
+
+DataProxy 实现了统一抽象 MQ Sink 模型，支持在标准 MessageQueueZoneSink 下方便灵活添加不同类型的 MQ 流向，默认支持 Pulsar Kafka 和 TubeMQ。本文将指导开发者如何扩展支持新的 MQ 类型。
```

Comment:
MQ Sink
->
MQ（Message Queue） Sink

Show the complete name at the first time.